Makefile

```diff
@@ -0,0 +1,15 @@
+PYTHON ?= python
+
+SPHINX_BUILD ?= sphinx-build
+
+.PHONY: man html
+man html:
+	$(SPHINX_BUILD) -a -b $@ doc build/sphinx/$@
+
+.PHONY: sdist wheel
+sdist wheel:
+	$(PYTHON) -m build --$@
+
+.PHONY: clean
+clean:
+	$(RM) -r build/sphinx doc/api
```

Comment:
Don't you need to define `RM`? Also, you want `-f` or it'll fail when dirs don't exist.

Reply:
Hmm, I didn't know that. I suppose it's fine.